src/sotoki/utils/preparation.py

```diff
@@ -213,7 +233,7 @@ def read_sub():
             main_id = get_id_in(main_line, field_index_in_main)
 
             # write main line to dest; removing tag end (/> -> >) and CRLF
-            dsth.write(main_line[:-4])
+            dsth.write(main_line[:-3])
```

Comment:
For all subsequent changes, the comment describing what's being done matches the magic number.
You're changing the numbers so please improve the comment or use variables instead